ckanext/cioos_theme/plugin.py

```diff
@@ -374,6 +377,8 @@ def before_index(self, data_dict):
             log.error("error:%s, keywords:%r", err, data_dict.get('keywords', '{}'))
             tags_dict = {"en": [], "fr": []}
 
+        data_dict['responsible_organizations'] = [x.get('organisation-name') for x in json.loads(data_dict.get('cited-responsible-party', '{}')) if x.get('role') in ['originator']]
```

Comment:
Minor but are we enforcing any limits on line lengths e.g. 100 chars? Often a good idea for reviewing PRs...